src/dmd/init.d

```diff
@@ -243,20 +202,15 @@ extern (C++) final class ArrayInitializer : Initializer
  */
 extern (C++) final class ExpInitializer : Initializer
 {
-    Expression exp;
     bool expandTuples;
+    Expression exp;
```

Comment:
It's there because it reduces the size of the struct by 8 bytes.

Reply:
Perhaps a dumb question, but which struct?

Reply:
The compiler has the opportunity to lay out:
```
ubyte bool exp
```
in 8 less bytes than:
```
ubyte exp bool
```
Whether it does or not is implementation defined (and since this is a C++ class, it is defined by the particular ABI).